src/DateTimeInvalid.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3;
+
+use InvalidArgumentException;
+
+final class DateTimeInvalid extends InvalidArgumentException
```

Comment:
Nit: maybe change this to `InvalidDateTime` to be more consistent with `InvalidArgumentException` and `InvalidAgeRangeException`?

Reply:
I went with `DateTimeInvalid` so it would be grouped together with `DateTimeFactory` in the list of files in the `src` directory. But if you want I can change it.

Reply:
No that is perfect!